test/modules/controls.js

```diff
@@ -19,6 +19,9 @@
             if (viewer && viewer.close) {
                 viewer.close();
             }
+            // if (viewer && viewer.destroy){
```

Comment:
I guess we don't need this?

Reply:
Yes, for this module the viewer isn't re-created for each test so I've left it as "close" and removed the commented code.